main.js

```diff
@@ -404,6 +405,10 @@ app.on('ready', function() {
 
 });
 
+ipcMain.on('set-badge', function(event, text) {
+  typeof text !== 'undefined' && app.dock.setBadge(String(text));
```

Comment:
This throws an error on Windows.

Cannot read property 'setBadge' of undefined.

You probably need to put a platform check in there.